wordpress-mobile/WordPress-iOS
# Re-add `newpost` url scheme

Fixes #11152 

This PR isn't yet finished. Here are the notes:
 - No support for format for the post. It just assumes HTML, basically
 - Doesn't have proper `x-callback -url` support yet
 - Doesn't support an image, either
 - Moves our url scheme code to swift. Another chunk of the app delegate converted 😀 

To test:
 - visit [this test site](http://heagy.com/urltest) (in simulator or a device with _this_ build)
 - tap the links to test creating a new post via the scheme
 - tap the links to test viewing stats or a specific post respectively to ensure those existing functions continue to work
 - login via Google to ensure _that_ still works
 - to test debug crashing you'd need to dig out our Crashlytics app key. Let me know if you want to do that 😜 

Update release notes:
